utils/githooks/pre-commit.d/30-yamllint.sh

```diff
@@ -14,4 +15,12 @@ then
 fi
 
 echo "Checking yaml formatting"
-yamllint --strict .
+targets=(
+    '*.yml'
+    '*.yaml'
+)
+
+files=${files:=$(git diff --diff-filter=ACMRTUXB --name-only --cached -- "${targets[@]}")}
+if [ -n "$files" ]; then
+    yamllint --strict $files
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]

Reply:
This is why I put the files into an array and then used "${files[@]}".  Not quoting is fragile.